stubs/ObjectStorage.stub

```diff
@@ -3,8 +3,8 @@ namespace TYPO3\CMS\Extbase\Persistence;
 
 /**
  * @template T
- * @implements \ArrayAccess<string, T>
- * @implements \Iterator<string, T>
+ * @implements \ArrayAccess<int|string, T>
+ * @implements \Iterator<int|string, T>
```

Comment:
`ObjectStorage::key()`, that is the implementation for `\Iterator::key()` does return a string.
So `int` is not correct here as `Iterator` will use `int` now with this annotation. So we can't do it like that.
Internally this iterator does also use a string and not an int. So i would suggest to keep this with string and just change the annotations of the methods that have a deviating behavior